src/main/java/org/example/pages/RegistrationPage.java

```diff
@@ -11,6 +12,14 @@ public class RegistrationPage extends BasePage {
     private static final By PASSWORD_INPUT_LOCATOR = By.xpath("//input[@name='password1']");
     private static final By REPEAT_PASSWORD_INPUT_LOCATOR = By.xpath("//input[@name='password2']");
     private static final By REGISTRATION_BUTTON_LOCATOR = By.xpath("//button[contains(text(), 'Register User')]");
+    private static final By ALREADY_HAVE_AN_ACCOUNT_LOCATOR = By.xpath("//a[contains(text(), 'Login!')]");
+    private static final By CREATE_A_USER_TITLE_LOCATOR = By.xpath("//h1[text()='Create a User!']");
+    private static final By MORE_150_OVER_CHARACTERS_ERROR_LOCATOR = By.xpath("//li[text()='Ensure this value has at most 150 characters (it has 151).']");
+    private static final By FORBIDDEN_CHARACTERS_ERROR_LOCATOR = By.xpath("//li[text()='Enter a valid username. This value may contain only letters, numbers, and @/./+/-/_ characters.']");
+    private static final By PASSWORD_AND_REPEATPASSWORD_DIFFERENT_ERROR_LOCATOR = By.xpath("//li[text()='The two password fields didn’t match.']");
+    private static final By SHORT_PASSWORD_ERROR_LOCATOR = By.xpath("//li[text()='This password is too short. It must contain at least 8 characters.']");
+    private static final By PASSWORD_ONLY_DIGITS_ERROR_LOCATOR = By.xpath("//li[text()='This password is entirely numeric.']");
+    private static final By PASSWORD_AND_USERNAME_ARE_SIMILAR_ERROR_LOCATOR = By.xpath("//li[text()='The password is too similar to the username.']");
```

Comment:
What about CREDENTIALS instead of PASSWORD_AND_USERNAME?

Reply:
I'd say credentials is a too broad term here. Current name shows what this error is about.